### ISSUE TITLE
[logging] Update security logs with leveled logs

### ISSUE THREAD

**Issue by gregnazario** [libra/libra#5612]
### Overview
Let's migrate send_struct_log! to use leveled macros, and remove `crit!` logs

### Tasks
- [ ] Remove any reference to `.warning()` or `.critical()`
- [ ] Change `send_struct_log!` to `sl_info!`, `sl_error!`, etc... accordingly similar to the text log macros
- [ ] replace any `crit!` with `error!`

**Comment by gregnazario** [comment]
@mimoo  :) if you could take some time to appropriately fix these logs, that would be great!

I'm tracking all of these in the logging project, to ensure I know how things are working out.

**Comment by mimoo** [comment]
wait, no more send_struct_log!? what's the rationale behind that : o ? Also why no more critical?

**Comment by gregnazario** [comment]
In some macro magic, in order to decrease the amount of logs being logged into the structured logs, moving to a simple macro system that people are familiar with `warn!` or `info!` to define the levels at the time it's written rather than having an optional level system.  I also considered just putting it in the constructor, but I decided that this was simpler from a usability standpoint.  My goal here is to make it such that we can move all text logs to struct logs at some point.  The complete picture of that isn't quite complete.

Critical had a bunch of TODOs around it because they weren't supported by the underlying text log system.  We had moved from a system that had `critical` to one that didn't, so I consolidated it to be the same.  If there's some particular reason we need `critical` over `error` plus filters then let me know.  But, how I see it we should be able to make an alarm on `security` and `error` for your use case.
